patches/readable-stream+2.3.6.patch

```diff
@@ -0,0 +1,103 @@
+diff --git a/node_modules/readable-stream/lib/_stream_duplex.js b/node_modules/readable-stream/lib/_stream_duplex.js
```

Comment:
We don't use v2.3.6 of readable-stream anymore. We use v2.3.7 which includes this patch: https://github.com/nodejs/readable-stream/pull/423
So maybe we don't need this patch at all anymore?

Reply:
this patch should be updated or removed